scribus/plugins/scriptplugin/scriptplugin.cpp

```diff
@@ -784,6 +785,12 @@ void initscribus(ScribusMainWindow *mainWin)
 	PyDict_SetItemString(d, const_cast<char*>("PAGE_2"), Py_BuildValue(const_cast<char*>("i"), 1));
 	PyDict_SetItemString(d, const_cast<char*>("PAGE_3"), Py_BuildValue(const_cast<char*>("i"), 2));
 	PyDict_SetItemString(d, const_cast<char*>("PAGE_4"), Py_BuildValue(const_cast<char*>("i"), 3));
+	// tab alignment
+	PyDict_SetItemString(d, const_cast<char*>("TAB_LEFT"), Py_BuildValue(const_cast<char*>("i"), ParagraphStyle::AlignmentType::LeftAligned));
```

Comment:
those are not the right values for the alignment constants.

i could not find any in `styles/paragraphstyle.h`... scribus seems to be using plain integers... (but you should defined constants for the scripter... of course : - )

the values are:

- Left
- Right
- Period
- Comma
- Center

which are probably integers from 0 to 4...